org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -8,14 +8,13 @@
 var request = require('request-promise');
 var OpenT2T = require('opent2t').OpenT2T;
 var Crypto = require('crypto');
-var accessTokenInfo = require('./common').accessTokenInfo;
 
 /**
 * This translator class implements the "Hub" interface.
 */
 class Translator {
-    constructor(accessToken) {
-        this._accessToken = accessToken;
+    constructor(authTokens) {
+        this._authTokens = authTokens;
```

Comment:
I am assuming you actually ran the CLI and ensured that was the case?  If not please do.. ! thanks

---
In reply to: [101102384](https://github.com/openT2T/translators/pull/160#discussion_r101102384) [](ancestors = 101102384,101099112)